src/composer.json

```diff
@@ -39,7 +39,7 @@
     
     "config": {
         "platform": {
-            "php": "5.3.3"
```

Comment:
Would it be an option to tag the commit 414a41c as release 3.1?

Or we could revert this PR and wait for packaging to be working